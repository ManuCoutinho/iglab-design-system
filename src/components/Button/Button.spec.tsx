import { screen, render } from '@testing-library/react'
import { Button } from '.'

describe('<Button/>', () => {
  it('should render a Button component correctly', () => {
    render(<Button>Text</Button>)
    expect(screen.getByRole('button')).toBeInTheDocument()
  })
  it('match to snapshot', () => {
    const { container } = render(<Button>Text</Button>)
    expect(container.firstChild).toMatchSnapshot()
  })
})
