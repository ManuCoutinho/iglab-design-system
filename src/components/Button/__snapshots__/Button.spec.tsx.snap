// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button/> match to snapshot 1`] = `
<button
  class="text-black font-sans font-semibold py-3 px-4 bg-cyan-500 rounded text-sm w-full hover:bg-cyan-300 transition-colors delay-75 focus:ring-2 focus:ring-cyan-500 focus:outline-cyan-300 "
>
  Text
</button>
`;
